test/e2e/windows/framework.go

```diff
@@ -16,8 +16,23 @@ limitations under the License.
 
 package windows
 
-import "github.com/onsi/ginkgo"
+import (
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+)
+
+// it seems we can't use k8s.io/kubernetes/test/utils/image.GetPauseImageName() as it does not
+// contain a windows image?
+const winPauseImageName = "kubeletwin/pause:latest"
```

Comment:
Not a fan of this being hardcoded here.

Also, the pause image here ( the one returned by GetPauseImageName() ): https://github.com/kubernetes/kubernetes/blob/b67126e350fdba6d89a0ee64310cac7db14e33d3/test/utils/image/manifest.go#L128 can be a windows image. 

By passing this env var: https://github.com/kubernetes/kubernetes/blob/b67126e350fdba6d89a0ee64310cac7db14e33d3/test/utils/image/manifest.go#L66 

with a file containing the registry names for windows images, the pause image should be fine.

In the upstream tests we use these registries: https://github.com/kubernetes-sigs/windows-testing/blob/master/images/image-repo-list

In fact, all the dockerfiles + documentation of the images we use in upstream e2e windows tests is here:

https://github.com/kubernetes-sigs/windows-testing

Reply:
👍